src/components/FiltersDrawer/index.tsx

```diff
@@ -74,11 +75,17 @@ const FiltersDrawer: React.FC<FiltersDrawerProps> = ({
       { filterName: filter.name, fieldName, fieldValue },
       "user"
     );
+    sendEvent({
+      name: "Edited Filter",
+      before: filter,
+      after: { ...filter, [fieldName]: fieldValue },
+    });
```

Comment:
maybe you can do this:
```suggestion
    sendEvent({
      name: "Edited Filter",
      before: filter,
      after: newFilters[idxToReplace],
    });
```